### PR TITLE
Suppressing color output with --color false, closes #105

### LINKED ISSUE #105: `--color false` CLI option does not work
`dredd apiary.apib http://localhost:3000 --color false` does not changes anything, it still outputs to terminal with ANSI colors.

### PR DESCRIPTION
**Pull request by netmilk**

### CODE REVIEW

File: test/integration/cli-test.coffee [hunk 1]
```diff
@@ -46,7 +46,7 @@ describe "Command line interface", () ->
     it 'should print error message to stderr', () ->
       assert.include stderr, 'Error: ENOENT, open'
 
-  describe "Arguments with existing bleuprint and responding server", () ->
+  describe "Arguments with existing blueprint and responding server", () ->
```
Comment (kuba-kubula) [inline test/integration/cli-test.coffee]: :+1:

File: src/apply-configuration.coffee [hunk 2]
```diff
@@ -45,6 +45,12 @@ applyConfiguration = (config) ->
   configuration.options.header = coerceToArray(configuration.options.header)
   configuration.options.method = coerceToArray(configuration.options.method)
 
+  #coerce color to bool
+  if configuration.options.color == 'false'
+    configuration.options.color = false
+  else if configuration.options.color == 'true'
+    configuration.options.color = true
+
```
Comment (XVincentX) [inline src/apply-configuration.coffee]: configuration.options.color = (configuration.options.color isnt false)

Comment (XVincentX) [inline src/apply-configuration.coffee]: It could be written in a single statement

Comment (netmilk) [inline src/apply-configuration.coffee]: Thx, but I consider it readable enough.  :)

**Pull request merged**